core/templates/pages/partnerships-page/partnerships-page.component.html

```diff
@@ -661,6 +658,11 @@ <h3 class="oppia-partnerships-h3">{{ 'I18N_PARTNERSHIPS_PAGE_FREQUENTLY_ASKED' |
     .carousel-control-next-icon {
       margin-right: -5em;
     }
+
+    .btn {
```

Comment:
Normally for generic class names we try to prefix them with something more specific so that their effects don't spill outside the component they're defined in.

Reply:
Done!